R/survey_summary_US_West_Coast.R

```diff
@@ -0,0 +1,55 @@
+# Explorations of US West Coast trawl surveys
+
+# surveys to consider
+survey_names <- c("Triennial", "AFSC.Slope", "NWFSC.Slope", "NWFSC.Combo")
+# extract data and bind into single dataframe
+catch <- NULL
+for (survey in survey_names) {
+  catch <- rbind(catch, 
+    nwfscSurvey::pull_catch(
+      common_name = "Pacific spiny dogfish", # random species
+      survey = survey
+    )
```

Comment:
Should we add the `years` argument here?  The figure includes 1977 Triennial data, but since this year is not recommended for use in assessments (at least not currently), this could cause some confusion.